Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -11400,6 +11400,22 @@ bool CStaticFunctionDefinitions::GetAccountSerial(CAccount* pAccount, SString& s
     return bRegistered;
 }
 
+
+bool CStaticFunctionDefinitions::SetAccountSerial(SString strName, SString strSerial)
```

Comment:
SString is no longer used, please use std::string instead

Reply:
adjusted 62ebf06